### PR TITLE
Pushing Data Prepper 2.3.0 & announcement blog live

### PR DESCRIPTION
**Pull request by krisfreedain**
### Description
Data Prepper 2.3.0 release artifacts
Announcing Data Prepper 2.3.0
 
### Issues Resolved
https://github.com/opensearch-project/project-website/pull/1651
https://github.com/opensearch-project/project-website/pull/1631

### Check List
- [x] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Review by nateynateynate — APPROVED** [review]

File: _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+meta_keywords: Data Prepper expressions, tail sampling in Data Prepper, Amazon S3 Sink
+meta_description: Learn how Data Prepper 2.3.0 improves the ability to process complex expressions, supports event tagging, enhances the add entries processor, and supports Amazon S3 sink. Download it now.
+---
+
+Data Prepper 2.3.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release introduces a number of changes that improve Data Prepper’s ability to process complex expressions with functions, arithmetic operations, and string operations.
+
+## Enhancements to Data Prepper expressions
+
+Data Prepper 2.3 supports using functions in expressions. A list of supported functions is available in the [Expression syntax](https://opensearch.org/docs/latest/data-prepper/pipelines/expression-syntax/) documentation.
```
Comment (vagimeli) [inline _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md:19]: Either use 2.3.0 or 2.3 consistently throughout the blog.

File: _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+
+String expressions evaluate to a result of string type. The string concatenation operator is supported, and it uses functions, JSON pointers, or literals as operands. For example, the following expression concatenates the `message1` field of type string in the event with the `message2` field in the event and appends "suffix" to it:
+
+```
+/message1 + /message2 + "suffix"
+```
+
+## Event tagging
+
+Data Prepper 2.3 supports tagging events while using the `grok` processor. Events can be tagged optionally be tagged using the following configuration:
```
Comment (vagimeli) [inline _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md:45]: Delete the redundant "be tagged" after "optionally" in the second sentence.

File: _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+       entries:
+         - key: "request_len"
+           value_expression: "length(/request)"
+```
+
+This configuration adds an entry with the key `request_len` with a value equal to the length of the request field in the key. The value expression can be any of the expressions supported. See [Supported operators](https://github.com/opensearch-project/data-prepper/blob/main/docs/expression_syntax.md) for more details.
+
+### Setting event metadata keys
+
+The `add_entries` processor can add entries in the event’s metadata instead in the event itself. The following example shows an `add_entries` processor configuration for adding an entry to metadata:
```
Comment (vagimeli) [inline _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md:77]: Add "of" after the word "instead" in the first sentence.

File: _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md [hunk 4]
```diff
@@ -0,0 +1,157 @@
...
+
+
+## Amazon S3 sink
+
+Data Prepper now supports saving data to Amazon Simple Storage Service (Amazon S3) sinks as Newline delimited JSON (ndjson). Amazon S3 is a popular choice for storing large volumes of data reliably and cost effectively.
+
+Ingesting data into Amazon S3 offers many possibilities for your data pipelines, including some of the following:
+
+* Noisy or uninteresting data can be routed into Amazon S3 and not to OpenSearch in order to reduce load on your OpenSearch cluster. This can help you save on compute and storage costs.
+* Ingestion data into Amazon S3 to have normalized data on-hand that you can use for future processing.
```
Comment (vagimeli) [inline _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md:96]: Please clarify the second bullet. For example, "Ingesting data into Amazon S3 normalizes the data you have on-hand for future processing."

File: _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md [hunk 5]
```diff
@@ -0,0 +1,157 @@
...
+
+```
+
+## Getting started
+
+* To download Data Prepper, see the [OpenSearch downloads](https://opensearch.org/downloads.html) page.
+* For instructions on how to get started with Data Prepper, see [Getting started with Data Prepper](https://opensearch.org/docs/latest/data-prepper/getting-started/).
+* To learn more about the work in progress for Data Prepper 2.4, see the [Data Prepper roadmap](https://github.com/opensearch-project/data-prepper/projects/1).
+
+
```
Comment (vagimeli) [inline _posts/2023-05-25-Announcing-Data-Prepper-2.3.0.md:139]: Delete extra line.

**Pull request merged**